### PR TITLE
release notes 2.0.3.8

### PR DESCRIPTION
**Pull request by nathan-px**
For 2.1.1.1 and 2.0.3.8 release notes accompanying 2.1.2

### CODE REVIEW

**Comment by nathan-px** [comment]
@prabirpaul https://portworx.atlassian.net/browse/PWX-9486 is stil outstanding. Can you help me understand the issue?

questions I have:
What was happening that was the reason for this change?
What does this change do?
How does this change improve things going forward?

File: content/reference/release-notes/_index.md [hunk 1]
```diff
@@ -214,6 +256,14 @@ April 19, 2019
 
 * PWX-8470: ASG: CLI does not update metadata device name, if after restart device name changes
 
+## 2.0.3.8
+
+### Key Fixes
+
+**PWX-9486** - Changes to server runtime configuration.
```
Comment (vilasd) [inline content/reference/release-notes/_index.md]: Its not server ===> Portworx

File: content/reference/release-notes/_index.md [hunk 2]
```diff
@@ -146,6 +146,48 @@ _User Impact:_ Portworx fails to attach a cloud drive with the following error m
...
 
 
+
+## 2.1.1.1
+
+July 25, 2019
+
+### Key Fixes:
+
+**PWX-9058** - Offline Autoscaling Group (ASG) storageless nodes are unable to be decomissioned after the cluster is scaled up.
```
Comment (vilasd) [inline content/reference/release-notes/_index.md]: This should be same as the one in 2.1.2 release notes

File: content/reference/release-notes/_index.md [hunk 3]
```diff
@@ -146,6 +146,48 @@ _User Impact:_ Portworx fails to attach a cloud drive with the following error m
...
+
+_User Impact:_ With 2.1.1.1, old KVDB nodes created prior to failover will properly update their host IP addresses.
+
+**PWX-8791** - Output from the `pxctl -j service kvdb endpoints` and `pxctl -j service kvdb members` commands is in plaintext.
+
+_User Impact:_ With 2.1.1.1, these commands now output JSON.
+
+**PWX-8788** - An internal KVDB installation fails in a non-Kubernetes environment.
+
+_User Impact:_ Previously, the `/etc/host` container may have been mounted as read-only, preventing successful installation. With 2.1.1.1, this container is now mounted with the proper read/write permissions.
```
Comment (vilasd) [inline content/reference/release-notes/_index.md]: With 2.1.1.1 , the mount inside the container is now mounted with proper read/write permissions

File: content/reference/release-notes/_index.md (hunk 2 above)
Comment (nathan-px) [reply content/reference/release-notes/_index.md]: I didn't find 9058 elsewhere in the release notes.

**Pull request merged**